distsql/select_result.go

```diff
@@ -106,6 +107,34 @@ func (r *selectResult) Fetch(ctx context.Context) {
 }
 
 func (r *selectResult) fetchResp(ctx context.Context) error {
+	defer func() {
+		if r.stats != nil {
+			coprCacheHistogramHit.Observe(float64(r.stats.CoprCacheHitNum))
+			coprCacheHistogramMiss.Observe(float64(len(r.stats.copRespTime) - int(r.stats.CoprCacheHitNum)))
```

Comment:
let's add GTE0 in case of different execution path between CoprCache and PlanExec

```
telemetry.CurrentCoprCacheHitRatioGTE0Count.Inc()
```